### PR TITLE
3845 datacite xml

### PR DESCRIPTION
**Pull request by pdurbin**
## Related Issues

- connects to #3845 Dataset (metadata) Title with '&' (ampersand) in the text causes 'Publish' to fail.

## Pull Request Checklist

- [ ] Unit [tests][] completed
- [ ] Integration [tests][]: None
- [ ] Deployment requirements, [SQL updates][], [Solr updates][], etc.: None
- [ ] [Documentation][docs] completed
- [ ] Merged latest from "develop" [branch][] and resolved conflicts

[tests]: http://guides.dataverse.org/en/latest/developers/testing.html
[SQL updates]: https://github.com/IQSS/dataverse/tree/develop/scripts/database/upgrades
[Solr updates]: https://github.com/IQSS/dataverse/blob/develop/conf/solr/4.6.0/schema.xml
[docs]: http://guides.dataverse.org/en/latest/developers/documentation.html
[branch]: http://guides.dataverse.org/en/latest/developers/branching-strategy.html

### CODE REVIEW

File: src/main/java/edu/harvard/iq/dataverse/DatasetPage.java [hunk 1]
```diff
@@ -3707,10 +3707,10 @@ public String getTitle() {
     /**
      * dataset description
      *
-     * @return title of workingVersion
+     * @return description of workingVersion
      */
     public String getDescription() {
-        assert (null != workingVersion);
+        // TODO: Switch to `getDescriptionPlainText()` when we are ready to QA this.
```
Comment (matthew-a-dunlap) [inline src/main/java/edu/harvard/iq/dataverse/DatasetPage.java]: Why?

Comment (pdurbin) [inline src/main/java/edu/harvard/iq/dataverse/DatasetPage.java]: @pameyer wrote this code this code for #1393 and said he would prefer plain text. I don't think we want to re-test all that Zotero stuff right now though.

File: src/main/java/edu/harvard/iq/dataverse/util/xml/XmlValidator.java [hunk 2]
```diff
@@ -2,24 +2,49 @@
...
+        SchemaFactory schemaFactory = SchemaFactory.newInstance(XMLConstants.W3C_XML_SCHEMA_NS_URI);
+        Schema schema = schemaFactory.newSchema(schemaToValidateAgainst);
+        Validator validator = schema.newValidator();
+        try {
+            validator.validate(xmlFile);
+            logger.info(xmlFile.getSystemId() + " is valid");
+            return true;
+        } catch (SAXException ex) {
+            System.out.print(ex.getMessage());
+            System.out.print(ex.getLocalizedMessage());
```
Comment (oscardssmith) [inline src/main/java/edu/harvard/iq/dataverse/util/xml/XmlValidator.java]: Remove `System.out.print`

File: src/main/java/edu/harvard/iq/dataverse/util/xml/XmlValidator.java [hunk 3]
```diff
@@ -41,4 +66,35 @@ public static boolean validateXml(String fileToValidate, String schemaToValidate
...
+            logger.info("XML is not well formed: " + ex);
+            return false;
+        }
+
+    }
+
+    public static class SimpleErrorHandler implements ErrorHandler {
+
+        public void warning(SAXParseException e) throws SAXException {
+            System.out.println(e.getMessage());
```
Comment (oscardssmith) [inline src/main/java/edu/harvard/iq/dataverse/util/xml/XmlValidator.java]: and here

**Comment by oscardssmith** [comment]
Overall these changes look reasonable, but someone else should take a look, as I don't know XML that well.

File: src/main/java/edu/harvard/iq/dataverse/util/xml/XmlValidator.java (hunk 3 above)
Comment (pdurbin) [inline src/main/java/edu/harvard/iq/dataverse/util/xml/XmlValidator.java]: Removed in fb8b597

File: src/main/java/edu/harvard/iq/dataverse/util/xml/XmlValidator.java (hunk 2 above)
Comment (pdurbin) [inline src/main/java/edu/harvard/iq/dataverse/util/xml/XmlValidator.java]: Removed in 825332b

**Pull request merged**